pkg/plugin/conftest/plugin_test.go

```diff
@@ -1,4 +1,4 @@
-package conftest_test
+package conftest
```

Comment:
Why you change the package name? It's a good practice to use `conftest_test` to make sure we are testing public APIs exposed by the `conftest` package, not private methods or implementation details.